lochness/mediaflux/__init__.py

```diff
@@ -142,14 +142,14 @@ def sync_module(Lochness: 'lochness.config',
                             subj_dir = subject.protected_folder \
                                 if protect else subject.general_folder
 
-                            mf_local = str(tree.get(datatype,
-                                                subj_dir,
-                                                processed=processed,
-                                                BIDS=Lochness['BIDS']) / \
-                                                        subpath.parent)
+                            mf_local = tree.get(datatype,
+                                            subj_dir,
+                                            processed=processed,
+                                            BIDS=Lochness['BIDS']) / \
+                                                    subpath.parent
 
-                            mf_local = str(Path(mf_local) / prod['out_dir']) \
-                                    if 'out_dir' in prod else mf_local
+                            mf_local = str(mf_local / prod['out_dir'] \
```

Comment:
* Get rid of `Path` as well
* Apply `str` to final `mf_local`

Reply:
is `mf_local` already Path by this point if we follow https://github.com/AMP-SCZ/lochness/pull/69#discussion_r786300908 ?

Reply:
Yes it is.

Reply:
You have already answered why:

> which works with /